### PR TITLE
SQL-2382: Fetch mongosqltranslate and add to msi, dmg, and linux tar

### PR DESCRIPTION
**Pull request by pmeredit**
I guess we can't really test this until we have the bucket.

### CODE REVIEW

**Review by mattChiaravalloti — APPROVED** [review]
Looks good once everything that's partially blocking is resolved.

File: evergreen.yml [hunk 1]
```diff
@@ -63,17 +73,17 @@ functions:
...
           export MSI_FILENAME="mongoodbc-eap-$RELEASE_VERSION.msi"
           export UBUNTU_FILENAME="mongoodbc-eap-$RELEASE_VERSION.tar.gz"
 
           cat <<EOT > expansions.yml
           RELEASE_VERSION: "$RELEASE_VERSION"
-          LIBMONGOTRANSLATE_VERSION: "$LIBMONGOTRANSLATE_VERSION"
+          LIBMONGOSQLTRANSLATE_VERSION: "$LIBMONGOSQLTRANSLATE_VERSION"
           MSI_FILENAME: "$MSI_FILENAME"
           UBUNTU_FILENAME: "$UBUNTU_FILENAME"
           WINDOWS_INSTALLER_PATH: "mongosql-odbc-driver/windows/$RELEASE_VERSION-lib$LIBMONGOTRANSLATE_VERSION/release/${MSI_FILENAME}"
```
Comment (nbagnard) [inline evergreen.yml:89]:
```suggestion
          WINDOWS_INSTALLER_PATH: "mongosql-odbc-driver/windows/$RELEASE_VERSION-lib$LIBMONGOSQLTRANSLATE_VERSION/release/${MSI_FILENAME}"
```

**Review by nbagnard — APPROVED** [review]
I think you missed one of the rename, but otherwise it looks good to me

File: evergreen.yml (hunk 1 above)
Comment (pmeredit) [reply evergreen.yml:89]: Oh wow, that should be impossible. Wonder how that happened.

**Comment by pmeredit** [comment]
Wix works locally, so I think this is probably good for now.

**Pull request merged**